dist/profile/templates/confluence/vhost.conf

```
@@ -179,6 +179,8 @@ RewriteRule "^/display/JENKINS/ArtifactDeployer\+Plugin$" "https://plugins.jenki
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
 RewriteRule "^/display/JENKINS/Artifactory\+Plugin$" "https://plugins.jenkins.io/artifactory" [NC,L,QSA,R=301]
 RewriteCond %{HTTP_USER_AGENT} !^jenkins-wiki-exporter/(.*)$
+RewriteRule "^/display/JENKINS/Artifactory\+-\+Working\+With\+Pipeline\+Jobs\+in\+Jenkins#" "https://plugins.jenkins.io/workflow-aggregator" [NC,L,QSA,R=301]
```

Sorry that I missed another case and grateful that the CI job detected the issue (by a happy accident).  We don't want the trailing '#' character on the redirect source since we want to redirect the URL even if it does not include that trailing '#' character.
```suggestion
RewriteRule "^/display/JENKINS/Artifactory\+-\+Working\+With\+Pipeline\+Jobs\+in\+Jenkins" "https://plugins.jenkins.io/workflow-aggregator" [NC,L,QSA,R=301]
```